README.adoc

```diff
@@ -1,41 +0,0 @@
-== Client libraries for 10Duke Identity and Entitlement

```

Comment:
asciidoc converted to markdown so that the package readme files can be used as on nuget.org.

This file was converted so that there is only one format for the README documentation files.